src/ApiCompat/Microsoft.DotNet.ApiCompat.Tool/Program.cs

```diff
@@ -42,7 +48,7 @@ static int Main(string[] args)
             };
             Option<MessageImportance> verbosityOption = new(new string[] { "--verbosity", "-v" },
                 "Controls the log level verbosity. Allowed values are high, normal, and low.");
-            verbosityOption.SetDefaultValue(MessageImportance.High);
+            verbosityOption.SetDefaultValue(MessageImportance.Normal);
```

Comment:
Unrelated change. I thought that we already downgraded the verbosity for the CLI, but apparently not. This should not be high as we want the summary confirmation message to be printed when invoking the CLI tool.